src/main/java/bio/terra/service/dataset/flight/create/DatasetCreateFlight.java

```diff
@@ -29,6 +30,7 @@ public DatasetCreateFlight(FlightMap inputParameters, Object applicationContext)
 
         // get the required daos and services to pass into the steps
         ApplicationContext appContext = (ApplicationContext) applicationContext;
+        BufferService bufferService = (BufferService) appContext.getBean("bufferService");
```

Comment:
Can this use the same style as Muscles' PR? https://github.com/DataBiosphere/jade-data-repo/pull/987